app/Models/Foundation/Summit/Events/Presentations/Materials/PresentationMaterial.php

```diff
@@ -200,4 +200,45 @@ public function clearPresentation(){
     }
 
     use ScheduleEntity;
+
+    public function clone(): PresentationMaterial{
```

Comment:
this clone method should not be resolved statically on the base class
each class should implement its clone method and resolve it through polimorphism

Reply:
done